packages/office-ui-fabric-react/src/components/FocusZone/FocusZone.tsx

```diff
@@ -924,6 +935,208 @@ export class FocusZone extends React.Component<IFocusZoneProps> implements IFocu
 
     return false;
   }
+  private _movePageDown(): boolean {
```

Comment:
I think there's a larger question on if we think pageUp/pageDown should always update focus if/when it scrolls.

If we want this functionality to be introduced, it should probably be done in a way that it doesn't break/change the current behavior.

Also, fyi @joschect , @micahgodbolt , and @dzearing .

Reply:
i discussed the idea before with @dzearing and he likes it :)